controller/products.js

```diff
@@ -1,114 +1,123 @@
+const { sellerOffer } = require('../services/helper.js');
 
-  if (id && isNaN(Number(id))) {
+const getQuotes = (req, res) => {
+  if (req.query.productId && isNaN(Number(req.query.productId))) {
     return res.status(400).send('Bad Request.');
   }
 
-  let priceInfo;
-  let sellerInfo;
-
-  sellersModel.retrieveSellers()
-    .then((sellerData) => {
-      sellerInfo = sellerData;
-      return pricesModel.retrievePrices(id);
-    })
-    .then((productData) => {
-      priceInfo = productData;
-      return true;
-    })
-    .then(() => createQuotes(priceInfo, sellerInfo, req.query.sellerLimit))
-    .then((quoteData) => {
-      if (!quoteData.length) {
-        return res.status(404).send('Product Not Found.');
-      }
-      return res.send(quoteData);
-    })
-    .catch(() => res.status(500).send('Internal Server Error.'));
-};
+  db.query(`SELECT prices.price, prices.tax, prices.id, sellers.seller_name, sellers.return_policy, sellers.delivery_free, sellers.delivery_min, sellers.delivery_days, sellers.delivery_fee FROM prices, sellers WHERE prices.product_id = ${req.query.productId} AND prices.seller = sellers.id`, (err, result) => {
+    if (err) {
+      res.status(500).send(err);
+    } else if (result.length === 0) {
+      res.status(404).send('Product Not Found');
+    } else {
+      result.map((quote) => {
+        quote.offer = sellerOffer(quote);
+      });
+      res.send(result);
+    }
+  })
+}
 
 const addPrices = (req, res) => {
-  pricesModel.fetchPrices(req.body.productId)
-    .then((pricesRecord) => {
-      if (pricesRecord.length === 0) {
-        pricesModel.addPrices(req.body);
-      } else {
-        throw 'Product already exists';
-      }
-    })
-    .then(() => {
+  const values = [];
+
+  values.push(req.body.productId);
+  values.push(req.body.seller);
+  values.push(req.body.price);
+  values.push(req.body.tax);
+
+  db.query(`INSERT INTO prices (product_id, seller, price, tax) VALUES(?, (SELECT id FROM sellers WHERE seller_name = ?), ?, ?)`, values, (err) => {
+    if (err) {
+      res.status(500).send(err);
+    } else {
       res.sendStatus(200);
-    })
-    .catch((err) => {
-      res.status(404).send(err);
-    });
+    }
+  });
 };
 
 const addSeller = (req, res) => {
-  sellersModel.fetchSeller(req.body.id)
-    .then((sellerRecord) => {
-      if (sellerRecord.length === 0) {
-        sellersModel.addSeller(req.body);
-      } else {
-        throw 'Seller already exists';
-      }
-    })
-    .then(() => res.sendStatus(200))
-    .catch((err) => res.status(404).send(err));
+  const values = [];
+
+  values.push(req.body.seller);
+  values.push(req.body.returnPolicy);
+  values.push(req.body.deliveryFree);
+  values.push(req.body.deliveryMin);
+  values.push(req.body.deliveryDays);
+  values.push(req.body.deliveryFee);
+
+  db.query(`INSERT INTO sellers (seller_name,return_policy, delivery_free,delivery_min,delivery_days,delivery_fee) VALUES(?, ?, ?, ?, ?, ?)`, values, (err) => {
```

Comment:
Recovering your spacing in your column headers input list will help with readability